conf.py

```diff
@@ -31,7 +31,9 @@
 
 intersphinx_mapping = {
     'main_docs': ('https://docs.fire.ly', None),
-    'firely_net_sdk_docs': ('https://docs.fire.ly/projects/Firely-NET-SDK/', None),
+    'firely_terminal_docs': ('https://docs.fire.ly/projects/Firely-Terminal/', None),
+    'forge_docs': ('https://docs.fire.ly/projects/Forge/', None),
+    'firely_net_sdk_docs': ('https://docs.fire.ly/projects/Firely-NET-SDK/en/latest/', None),
```

Comment:
You have locale in url - is that intentional?

Reply:
Yes, because the targets where Intersphinx refers to live on https://docs.fire.ly/projects/Firely-NET-SDK/en/latest/objects.inv and not https://docs.fire.ly/projects/Firely-NET-SDK/objects.inv.

Only the SDK and Firely Server docs currently use multiple branches in Readthedocs.